Os/Posix/DefaultDirectory.cpp

```diff
@@ -0,0 +1,13 @@
+// ======================================================================
```

Comment:
Directory, FileSystem, and File should all be in one default file and one implementation, no?

Reply:
https://github.com/nasa/fprime/issues/2905